project/src/main/resources/db/migration/V1.1.2__student.sql

```diff
@@ -0,0 +1,14 @@
+create table STUDENTS (
+	student_id identity not null primary key, 
+	major varchar(50) not null,
+	gpa numeric(3,2),
+	first_name varchar(50) not null, 
+	last_name varchar(50) not null, 
+	email varchar(100)
+);
+
+insert into students (student_id, major, gpa, first_name, last_name, email)
+values ('123123', 'Computer Science', 4.00, "Joey", "Diaz", "joey.diaz@gmail.com")
+
+insert into students (student_id, major, gpa, first_name, last_name, email)
+values ('321321', 'Computer Science', 4.20, "Joe", "Rogan", "joe.rogan@gmail.com")
```

Comment:
Same here...single quote strings and semi-colon at the end of the query

insert into students (student_id, major, gpa, first_name, last_name, email)
values ('321321', 'Computer Science', 4.20, 'Joe', 'Rogan', 'joe.rogan@gmail.com');